README.md

```diff
@@ -17,24 +23,35 @@ your project.
+| [CLI](./cli)                                 | @aidanm3341, @lbulanti-ms, @willosborne | [![Build CLI](https://github.com/finos-labs/architecture-as-code/actions/workflows/cli-tests.yml/badge.svg)](https://github.com/finos-labs/architecture-as-code/actions/workflows/cli-tests.yml)                                                                                                                                                                                                                                                  |
+| [Spectral](./spectral)                       | @willosborne, @lbulanti-ms              | [![Validation of CALM Samples](https://github.com/finos-labs/architecture-as-code/actions/workflows/spectral-validation.yml/badge.svg)](https://github.com/finos-labs/architecture-as-code/actions/workflows/spectral-validation.yml)                                                                                                                                                                                                             |
+| [Translators](./translator)                  | @Budlee @matthewgardner @jpgough-ms     | [![Build Translators](https://github.com/finos-labs/architecture-as-code/actions/workflows/translator.yml/badge.svg)](https://github.com/finos-labs/architecture-as-code/actions/workflows/translator.yml)                                                                                                                                                                                                                                        |
+| [Visualizer](./visualizer)                   | @aidanm3341, @Budlee, @willosborne      |                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 
 ## Getting Involved
-Architecture as Code is part of the [DevOps Automation Special Interest Group](https://devops.finos.org/docs/working-groups/aasc/). Our Zoom meetups take place on the fourth Tuesday of every month, see [here](https://github.com/finos/devops-automation/issues?q=label%3Ameeting+label%3Aarchitecture-as-code+) for upcoming and previous meetings.
 
-Have an idea or feedback? [Raise an issue](https://github.com/finos-labs/architecture-as-code/issues/new/choose) in this repository.
+Architecture as Code is part of
```

Comment:
some quite strange wrapping going on here. is there an aggressive auto formatter making everything 80 chars or less wide?

Reply:
Yes